pkgs/development/python-modules/catboost/default.nix

```
@@ -41,6 +42,11 @@ buildPythonPackage rec {
   dontUseSetuptoolsCheck = true;
   pythonImportsCheck = [ "catboost" ];
 
+  passthru = [
+    # Do not update to catboost 1.1.x because the patch doesn't apply cleanly
+    skipBulkUpdate = true;
+  ];
```

it should be curly brackets:
```suggestion
  passthru = {
    # Do not update to catboost 1.1.x because the patch doesn't apply cleanly
    skipBulkUpdate = true;
  };
```

---

Sorry, I apparently wasn't awake.  Fixed now.